matchengine/internals/utilities/utilities.py

```diff
@@ -176,10 +176,10 @@ def get_sort_order(matchengine: MatchEngine, match_document: Dict) -> list:
         sort_array.append(sort_index)
 
     # If an idenfitifer is not a protocol id (e.g. 17-251) then skip replacing
-    identifier = match_document.get(matchengine.match_criteria_transform.trial_identifier, None)
-    if isinstance(identifier, ObjectId) or identifier is None:
-        pass
-    else:
-        sort_array.append(int(identifier.replace("-", "")))
+    # identifier = match_document.get(matchengine.match_criteria_transform.trial_identifier, None)
+    # if isinstance(identifier, ObjectId) or identifier is None:
+    #     pass
+    # else:
+    #     sort_array.append(int(identifier.replace("-", "")))
```

Comment:
The new version of ME will move this into TrialMatchDocumentCreator where it can be modified more easily.